features/image-transformations/resize-crop-and-other-transformations.md

```diff
@@ -875,3 +875,75 @@ By specifying this parameter in the file URL, ImageKit sets the `content-disposi
 Example - [https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?ik-attachment=true](https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?ik-attachment=true)
 
 ![](https://ik.imagekit.io/demo/medium_cafe_B1iTdD0C.jpg?ik-attachment=true)
+
+### Page - (pg)
+Delivers specified page, frame, or layer of a multi-page/frame/layer file, such as a PDF, animated image like GIF, or PSD. It needs to be used alongside the ik-thumbnail.jpg URL component.
+
+`ik-thumbnail.jpg` needs to be added after the file resource URL, like this:
+
+```markup
+https://ik.imagekit.io/demo/tr:pg-2/sample.pdf/ik-thumbnail.jpg
+````
+
+**List of supported multi-page/frame/layer files formats**
+* PDF
+* PSD
+* EPS
+* AI
+* GIF
+* Animated Webp
+* Animated PNG
+
+{% hint style="info" %}
+If the `pg` parameter is not provided, only `ik-thumbnail.jpg` is added after the file resource URL of the above mentioned formats. The first page, frame, or layer is extracted by default.
```

Comment:
This sentence looks malformed a little?
`If the `pg` parameter is not provided and only `ik-thumbnail.jpg` is added after the file resource URL of the above mentioned formats, then the first page, frame, or layer is extracted by default.`

Is this better?

Reply:
Done